src/libs/runtime-utils.js

```diff
@@ -63,13 +64,14 @@ const dataprocCost = (machineType, numInstances) => {
 export const runtimeConfigBaseCost = config => {
   const {
     cloudService, masterMachineType, masterDiskSize, numberOfWorkers, workerMachineType, workerDiskSize, bootDiskSize
-  } = normalizeRuntimeConfig(
-    config)
+  } = normalizeRuntimeConfig(config)
+
+  const isDataproc = cloudService === cloudServices.DATAPROC
 
   return _.sum([
     (masterDiskSize + numberOfWorkers * workerDiskSize) * storagePrice,
-    cloudService === cloudServices.DATAPROC && (dataprocCost(masterMachineType, 1) + dataprocCost(workerMachineType, numberOfWorkers)),
-    bootDiskSize * storagePrice
+    isDataproc && (dataprocCost(masterMachineType, 1) + dataprocCost(workerMachineType, numberOfWorkers)),
+    !isDataproc && (bootDiskSize * storagePrice)
```

Comment:
I think this was a bug. I've added the `!isDataproc` condition because for `Dataproc`, the line above (`(masterDiskSize + numberOfWorkers * workerDiskSize) * storagePrice`) already accounts for all disk costs.

Reply:
Would this be clearer?
```js
isDataproc ?
  (dataprocCost(masterMachineType, 1) + dataprocCost(workerMachineType, numberOfWorkers)) :
  (bootDiskSize * storagePrice)
```